qutebrowser/config/configtypes.py

```diff
@@ -1151,6 +1162,17 @@ class UserStyleSheet(File):
     def __init__(self):
         super().__init__(none_ok=True)
 
+    def transform(self, value):
+        if not value:
+            return None
+        path = super().transform(value)
+        path = os.path.expandvars(path)
+        if os.path.isabs(path):
```

Comment:
You could use the `os_path` mock to fake `os.path.exists` to return True:

```python
def test_transform_file(self, os_path):
    """Test transform with a filename."""
    os_path.exists.return_value = True
    ...
```

(see above for an explanation of the mocks)